client/src/modules/notification/Notification.scss

```diff
@@ -0,0 +1,14 @@
+.notification {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  padding: 50px 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-end;
+
+  &__wrapper {
+    max-width: 1440px;
```

Comment:
Where does this value come from? Maybe set it to 100%?

Reply:
same width as app wrapper that is declared to 1440px